src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/EmptyStatement/context/defaultContextConfig.xml

```diff
@@ -9,7 +9,7 @@
         <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
             <property name="file" value="${tp}/defaultContext.patch" />
             <property name="strategy" value="context" />
-            <property name="supportContextStrategyChecks" value="EmptyStatementCheck" />
+            <property name="neverSuppressedChecks" value="EmptyStatementCheck" />
```

Comment:
Correct me if I am wrong, in this test you want to check if `neverSuppressedChecks ` contains `EmptyStatementCheck` then violations are suppressed, otherwise user will receive no violations?

Reply:
@timurt, yes, `EmptyStatementCheck` belongs to https://github.com/checkstyle/patch-filters/issues/8#issuecomment-665565892, so should use `neverSuppressedChecks `.